### DISCUSSION TITLE
[WASM] [Mapsui.Uno.WinUI] ```Uno.WinUI``` 5.0.0-dev.*** build error

### DISCUSSION THREAD

**Discussion by limeniye** [Mapsui/Mapsui#2161] [General]
<h2>Description</h2>

An Wasm project doesn't build with Mapsui.Uno.WinUI and with latest Uno.WinUI version.

<h2>Steps</h2>

- Create **Uno Platform app** with net8.0.
- Update **Uno.WinUI** packages to 5.0.0-dev.*** versions for able to use the net8.0.
- Add **Mapsui.Uno.WinUI**
- Try to build Wasm platform

<h2>Expectations</h2>

I'am expecting successful build

<h2>Result</h2> 

```
2>C:\Users\mykyt\.nuget\packages\uno.wasm.bootstrap\8.0.0-dev.252\build\Uno.Wasm.Bootstrap.targets(204,3): error : System.Exception: Failed to run emscripten. More details are available in diagnostics logging mode or using the MSBuild /bl switch, see https://aka.platform.uno/msbuild-troubleshoot.
2>C:\Users\mykyt\.nuget\packages\uno.wasm.bootstrap\8.0.0-dev.252\build\Uno.Wasm.Bootstrap.targets(204,3): error :    at Uno.Wasm.Bootstrap.ShellTask_v141bdf179d88790116104b9a8c602bc694813ebc.RunPackager() in /_/src/Uno.Wasm.Bootstrap/ShellTask.cs:line 756
2>C:\Users\mykyt\.nuget\packages\uno.wasm.bootstrap\8.0.0-dev.252\build\Uno.Wasm.Bootstrap.targets(204,3): error :    at Uno.Wasm.Bootstrap.ShellTask_v141bdf179d88790116104b9a8c602bc694813ebc.Execute() in /_/src/Uno.Wasm.Bootstrap/ShellTask.cs:line 266
2>Done building project "UnoApp20.Wasm.csproj" -- FAILED.
```

<h2>Option</h2>

If I'll comment this package, build is successful.
```XML
<!--<PackageReference Include="Mapsui.Uno.WinUI" />-->
```

**Solution** [MapsuiWithNet8AndUnoWinUI.zip](https://github.com/Mapsui/Mapsui/files/12475855/MapsuiWithNet8AndUnoWinUI.zip)
**Visual Studio**: Visual Studio 17.8.0 Preview 1.0
**Net version**: 8.0.100-preview.7.23376.3

**Comment by inforithmics** [comment]
Follwing nugets need to be updated (So far I know)

```
<PackageReference Include="SkiaSharp.Views.Blazor" Version="2.88.5" />
<PackageReference Include="HarfBuzzSharp.NativeAssets.WebAssembly" Version="2.8.2.5" />
<PackageReference Include="SkiaSharp.HarfBuzz" Version="2.88.5" />
<PackageVersion Include="SQLitePCLRaw.bundle_green" Version="2.1.6" />
```

**Comment by inforithmics** [comment]
I guess to fully work Mapsui.Blazor needs to Target .NET 8 and Uno 5.0 But this is a breaking change because Uno 5.0 removed .NET 6 Framework support. And maybe other things. I'll have a look into that when .NET 8 is released as Release Candidate.

**Reply by limeniye** [reply]
Net 8 and Uno 5.0.** was released

**Reply by pauldendulk** [reply]
We will release the update in 5.0.0-preview.1
